### PR TITLE
:scream: The universal selector! :scream_cat:

### PR DESCRIPTION
**Pull request by gboone**
Turns off ligatures for all text. I think our 18F custom thingy is unaffected. Someone with better eyes for these kinds of things should verify that before merging. paging @jenniferthibault and @18F/design

### CODE REVIEW

File: assets/_sass/base/_variables.scss [hunk 1]
```diff
@@ -1,14 +1,13 @@
 // 18F Fonts
 $raleway: Raleway,'Helvetica Neue',Helvetica,Arial,sans-serif;
 $open-sans: 'Open Sans','Helvetica Neue',Helvetica,Arial,sans-serif;
-$brand: '18F Nimbus','Helvetica Neue',Helvetica,sans-serif;
-$nimbus: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
+$brand: 'Nimbus','Helvetica Neue',Helvetica,sans-serif;
```
Comment (jenniferthibault) [inline assets/_sass/base/_variables.scss]: @gboone is this now the original unedited Nimbus? Or the customized 18F Nimbus? It looks like the 18FNimbus files were removed

Comment (maya) [inline assets/_sass/base/_variables.scss]: i think the extra files changed are from the merge, instead of a rebase which would make the history cleaner.

Comment (gboone) [inline assets/_sass/base/_variables.scss]: No the extra files were just there cuz I forgot to remove them the first time I did this. :( see the title on #821 for more context.

**Comment by jenniferthibault** [comment]
As a temporary fix until we can properly figure out how to hack Nimbus' ligatures without breaking our numeric adjustments, 👍

**Comment by gboone** [comment]
Thanks to @maya for finding this issue and also to @jenniferthibault, @ericronne, @elainekamlley, and @rtwell for helping me fix it!

**Pull request merged**